sdc/tests/tests_perf/test_perf_base.py

```diff
@@ -6,19 +6,26 @@
 
 class TestBase(unittest.TestCase):
     iter_number = 5
+    test_results_class = TestResults
 
     @classmethod
-    def setUpClass(cls):
+    def create_test_results(cls):
```

Comment:
Maybe does it make sense to move the class method to `TestResults` class as class method?

Reply:
This method creates drivers and loads prev results. I think it should be decided outside the result class. So TestBase is good place for deciding it. I would like it method to stay here.